config.defaults.yaml

```diff
@@ -6,41 +6,152 @@ kowalski:
-        - name: eclipsing
-          children:
-            - name: EA
-            - name: EB
-            - name: EW
-        - name: flaring
-        - name: bogus
-    - name: ontological
-      children:
-        - name: pulsator
-          children:
-            - name: Delta Scuti
-            - name: Cepheid
-            - name: RR Lyrae
-            - name: LPV
-              children:
-                - name: Mira
-                - name: SRV
-        - name: binary star
-          children:
-            - name: W Uma
-            - name: Beta Lyr
-            - name: RS CVn
-        - name: AGN
-        - name: YSO
-        - name: CV
+  class: tds
+  name: time domain source type
+  subclasses:
+    - class: phenomenological
+      name: phenomenological
+      description: |
+        What can be said about an object given just its ZTF light curve.
+      subclasses:
+        - class: vnv
+          name: variable
+          description: |
+            Any ZTF source that shows detectable variability in its light curve
+            due to an astrophysical origin. This excludes variability due to blended photometry,
+            bright nearby stars, or any CCD artifact.
+        - class: pnp
+          name: periodic
+          description: |
+            The ZTF light curve features astrophysical periodic variability.
+            These are typically pulsators, rotating stars and (eclipsing) binaries.
+            This does not include semi-periodic or quasi-periodic variability. This
+            excludes variability due to a varying background (e.g. due to the moon),
+            or spurious periodic variability due to nearby bright stars or other artifacts.
+        - class: longt
+          name: long timescale
+          description: |
+            Any object which shows variability on timescales of 100 days. This can be a steady
+            increase/decrease in luminosity, e.g., AGN and CVs.
+            Long timescale period variable like Miras and the more
+            irregular semi-regular variables are generally included in this category.
+        - class: i
+          name: irregular
+          description: |
+            Objects which show irregular variability.
+            These are mostly accreting objects, AGN, CVs, and YSOs.
+        - class: e
+          name: eclipsing
+          description: |
+            Any source that shows eclipses in the light curve.
+            These are predominately eclipsing binaries.
+            Eclipsing planetisimals or planets with large rings would also
+            fall in this category.
+          subclasses:
+            - class: ea
+              name: detached binary
+            - class: eb
+              name: semi-detached binary
+            - class: ew
+              name: overcontact binary
+        - class: fla
+          name: flaring
+          description: |
+            Any ZTF light curve that shows flares, rapidly rising and fading events.
```

Comment:
Should M-dwarfs be mentioned by name?

Reply:
Sure, why not. To clarify: I just copy-pasted whatever we have in the paper-I (correcting the naming issues that were previously there).